internal/controlplane/common.go

```diff
@@ -27,7 +29,12 @@ import (
 	pb "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
 )
 
-const defaultProvider = oauth.Github
+const (
+	defaultProvider = oauth.Github
+	githubURL       = "https://github.com"
+)
+
+var validRepoSlugRe = regexp.MustCompile(`(?i)^[a-z0-9_]+\/[a-z0-9_]+$`)
```

Comment:
I think a repo name can also contain hyphens (`-`) and period (`.`).

Reply:
Nice, thanks for catching this